target/linux/realtek/image/rtl838x.mk

```diff
@@ -313,6 +313,8 @@ define Device/zyxel_gs1900-8
   $(Device/zyxel_gs1900)
   SOC := rtl8380
   DEVICE_MODEL := GS1900-8
+  DEVICE_ALT0_VARIANT := v1
+  DEVICE_ALT1_VARIANT := v2
```

Comment:
You need to specifiy the primary variant with `DEVICE_VARIANT := v1`. For the first alternative (`ALT0`) you also need to specify all three variables `DEVICE_ALT0_VENDOR`, `DEVICE_ALT0_MODEL`, `DEVICE_ALT0_VARIANT`.

You can use `make menuconfig` to check if the name shows up as intended.

Reply:
Will do. Thanks